addOns/ascanrulesAlpha/src/main/java/org/zaproxy/zap/extension/ascanrulesAlpha/HiddenFilesScanRule.java

```diff
@@ -270,6 +269,8 @@ private static String getOtherInfo(String type) {
                         new HiddenFile(
                                 hiddenFileObject.getString("path"),
                                 getList(hiddenFileObject.getJSONArray("content")),
+                                getList(hiddenFileObject.getJSONArray("not_content")),
```

Comment:
This could be optional (same for other fields not required), to not have to add all the properties to all entries.

Reply:
As discussed in IRC a full example should be documented in the help content as well.